### ISSUE TITLE
center of mass by entity with small main child entity 

### ISSUE THREAD

**Issue by Tackietacktack** [tsunamayo/Starship-EVO#5455]
### Short Description of the issue:
the center of mass is under the main child entity. main child entity is hinge, ship computer and yoke.
the center of mass is on the end of ship. when the ship gets gravity it will rotate about its back with the center of mass.

I have a solution for my ship. I built the main children from the ship to the front.
 So i don´t know if it is a great problem.

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 24w04a
- Graphics Card: 

### Additional information:

https://youtu.be/5sCObX1H9QM

screenshot main child about cockpit
![Screenshot (61)](https://github.com/tsunamayo/Starship-EVO/assets/97230091/a195d4e2-5581-4fdb-8222-e13c7058560d)
blueprint
[cockpit 15.zip](https://github.com/tsunamayo/Starship-EVO/files/14088954/cockpit.15.zip)

**Comment by BigBadKangaroo** [comment]
Hi there,

I had the same issue quite a while ago: #5105 
https://github.com/tsunamayo/Starship-EVO/assets/81495956/d6df25da-04db-4dae-8e36-e3836522f693

**Comment by Tackietacktack** [comment]
Exactly is the same

**Comment by ProPeach** [comment]
Yeah I've had ships do this too, it'd be great to see child entity mass distributed correctly
![image](https://github.com/tsunamayo/Starship-EVO/assets/27857711/cb85ef39-c31e-4954-b1cc-cc55b5f783dc)
